general-superstaq/general_superstaq/user_config_test.py

```diff
@@ -69,6 +69,15 @@ def test_update_user_balance(
     )
 
 
+def test_update_user_balance_limit() -> None:  # pylint: disable=missing-function-docstring
```

Comment:
I believe we already ignore these pragmas for the test files.
```suggestion
def test_update_user_balance_limit() -> None:
```